generators/server/templates/src/main/java/package/config/_DefaultProfileUtil.java

```diff
@@ -20,14 +18,7 @@
      * @param app the spring application
      */
     public static void addDefaultProfile(SpringApplication app) {
-        Map<String, Object> defProperties =  new HashMap<>();
-        /*
-        * The default profile to use when no other profiles are defined
-        * This cannot be set in the <code>application.yml</code> file.
-        * See https://github.com/spring-projects/spring-boot/issues/1219
-        */
-        defProperties.put(SPRING_PROFILE_DEFAULT, Constants.SPRING_PROFILE_DEVELOPMENT);
-        app.setDefaultProperties(defProperties);
+        app.setAdditionalProfiles(Constants.SPRING_PROFILE_DEVELOPMENT, Constants.SPRING_PROFILE_SWAGGER);
```

Comment:
take look at the logs from the travis build for any [prod profile](https://travis-ci.org/jhipster/generator-jhipster/jobs/159938444). you can see that its always running both `prod` and `dev` 
```
2016-09-14 17:15:50.253  INFO 7015 --- [           main] c.mycompany.myapp.TravisPsqlEsNoi18NApp  : Starting TravisPsqlEsNoi18NApp v0.0.1-SNAPSHOT on testing-gce-99b1c25b-38fb-4d0c-9d00-e84dbf0bf13c with PID 7015 (/home/travis/app/app.war started by travis in /home/travis/app)
2016-09-14 17:15:50.267  INFO 7015 --- [           main] c.mycompany.myapp.TravisPsqlEsNoi18NApp  : The following profiles are active: dev,swagger,prod
2016-09-14 17:15:59.250  WARN 7015 --- [           main] io.undertow.websockets.jsr               : UT026009: XNIO worker was not set on WebSocketDeploymentInfo, the default worker will be used
2016-09-14 17:15:59.253  WARN 7015 --- [           main] io.undertow.websockets.jsr               : UT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used
2016-09-14 17:15:59.882  INFO 7015 --- [           main] c.mycompany.myapp.config.WebConfigurer   : Web application configuration, using profiles: [dev, swagger, prod]
```

also this build has errors but its still marked as passed https://travis-ci.org/jhipster/generator-jhipster/jobs/159938442 I dont know why. @pascalgrimaud FYI